addons/test_mail/models/test_mail_models.py

```diff
@@ -133,7 +133,7 @@ class MailTestComposerMixin(models.Model):
 
     name = fields.Char('Name')
     author_id = fields.Many2one('res.partner')
-    description = fields.Html('Description', render_engine="qweb", render_options={"post_process": True})
+    description = fields.Html('Description', render_engine="qweb", render_options={"post_process": True}, sanitize=False)
```

Comment:
I think keeping a sanitize should be feasible. Even if for some reason we remove them in our fields, custom models / custom fields could be sanitized. It should still be able to render, unless it is not doable for some reason.